schematic/store/synapse.py

```diff
@@ -182,36 +186,30 @@ def __init__(
         self.project_scope = project_scope
         self.storageFileview = CONFIG.synapse_master_fileview_id
         self.manifest = CONFIG.synapse_manifest_basename
+        self.root_synapse_cache = "/root/.synapseCache"
```

Comment:
default the root of synapse cache to `/root/.synapseCache`. Note: this should not affect users when they use schematic locally because `self.root_synapse_cache` does not get used else where other than clearing .synapseCache on AWS.

Reply:
In the future, `root_synapse_cache` should probably be an argument when instantiating `SynapseStorage` class